adoc/chapters/programming_interface.adoc

```diff
@@ -17945,43 +17945,69 @@ device_event(___unspecified___)
 [[sec:atomic-references]]
 === Atomic references
 
-The SYCL specification provides atomic operations based on the library
-syntax from the next {cpp} specification. The set of supported orderings is
+The [code]#sycl::atomic_ref# class provides the ability to perform atomic
+operations in device code with a syntax similar to the {cpp} standard
+[code]#std::atomic_ref#.  The [code]#sycl::atomic_ref# class must not be used
+in host code.
```

Comment:
When we say host code here, do we mean application scope, or does this also extend to host tasks?

Reply:
The prohibition also extends to host tasks.  Therefore, I think "application scope" is not the right term here.

Reply:
That makes sense, just wanted to clarify, thanks.